.github/workflows/android.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           api-level: ${{ matrix.api-level }}
           target: ${{ matrix.target }}
           arch: x86_64
-          script: ./gradlew connectedCheck --no-daemon --stacktrace -PuseKsp=false
```

Comment:
oh I was suggesting adding `-DuseKsp` in addition to `-PuseKsp` - . The gradle files add dependencies based on the project flags so with the current setup, only kapt is running and ksp isn't even running in the tests - https://github.com/airbnb/Showkase/blob/master/sample/build.gradle#L67

Reply:
Ah, sorry. Totally my bad. I misunderstood that. Added it back in e2d2d67 :) Does this look correct?

Reply:
Perfect!